packages/passport/sdk-sample-app/src/components/zkevm/SignatureValidation/utils.ts

```diff
@@ -1,32 +1,22 @@
-import { Provider, TypedDataPayload } from '@imtbl/passport';
-import { ethers } from 'ethers';
-
 // https://eips.ethereum.org/EIPS/eip-1271#specification
 // EIP-1271 states that `isValidSignature` must return the following value if the signature is valid
-const ERC_1271_MAGIC_VALUE = '0x1626ba7e';
+import { ethers } from 'ethers';
+import { Provider } from '@imtbl/passport';
 
-export const isSignatureValid = async (
+export const ERC_1271_MAGIC_VALUE = '0x1626ba7e';
```

Comment:
Can we ensure that the `// EIP-1271 states that...` comment is above the `ERC_1271_MAGIC_VALUE` const please?